docs/user_stories/debug_failed_build.md

```diff
@@ -0,0 +1,23 @@
+### As a developer, with a PR that fails to build, I want to find the root cause so I can fix it and merges
+#### User Journey
+1. GitHub notifies user that build failed
```

Comment:
Maybe we can just say "user is notified that build failed"

I mean, it probably is github, but that itself might be worthy of creating a story around

Reply:
I was thinking that the story include the entry point and we could create stories with other entry points later on. Some entry points are more common (like GitHub PR) vs others in the future. This can help us prioritize between them